client/components/site-selector/style.scss

```diff
@@ -5,15 +5,13 @@
 .site-selector {
 	overflow: visible;
 	position: static;
-	pointer-events: auto;
 	border: none;
 	z-index: z-index( 'root', '.site-selector' );
 
 	&.is-large .search {
 		display: flex;
 		position: relative;
 		opacity: 1;
-		pointer-events: initial;
```

Comment:
Because of this removal, I'm unable to click on the site search bar after clicking Switch Site.  Elsewhere in this file there is a `pointer-events: none` rule for this element, and it is no longer being overridden.  https://github.com/Automattic/wp-calypso/pull/22507/files#diff-82484b636bfcea047c5bc590322bc7c4L104

Reply:
Ah, nice catch. I've no clue why we're toying with pointer-events here — I've removed it and everything seem to work as expected.